strimzi/strimzi-kafka-operator
# Integrate with Kafka Roller

### Type of change

_Select the type of your PR_

- Enhancement / new feature

### Description

* Fix existing altering config test
* Add desired config to TestingKafkaRoller
* Fix KafkaAgentClient call to node config endpoint
* Roll all unready pods, both broker and controllers before rolling the ready pods.
* Skip the admin "isController" check if the node is KRaft controller
* Fix Controller diff check when first changed config is not a controller config
* Add logs
* Add quorum check to be done before rolling controllers
* Update controller config list

### Checklist

_Please go through this checklist and make sure all applicable tasks have been done_

- [ ] Check RBAC rights for Kubernetes / OpenShift roles
- [x] Try your changes from Pod inside your Kubernetes and OpenShift cluster, not just locally
- [ ] Reference relevant issue(s) and close them after merging
- [ ] Supply screenshots for visual changes, such as Grafana dashboards
